### DISCUSSION TITLE
Replacement for out of stock SN74LVTH125PWR? and discontinued LED

### DISCUSSION THREAD

**Discussion by Original-Dan** [BlueSCSI/BlueSCSI-v2#113] [Build it Yourself]
hi just trying to order bluescsi via jlcpcb, on the desktop version the [Texas Instruments SN74LVTH125PWR](https://www.lcsc.com/product-detail/74-Series_TI_SN74LVTH125PWR_SN74LVTH125PWR_C7042.html)
is out of stock, just wanted to double check with folks here if this part: [onsemi 74LVTH125MTCX](https://www.lcsc.com/product-detail/Buffer-Driver-Transceiver_onsemi-74LVTH125MTCX_C903379.html)
is a compatible replacement before i finish the order? as I'm getting the smd assembly done on their end, the chip appears to match up size/spec wise but I'm not an expert with components so i may have overlooked something

also the blue LEDs on desktop/external versions are discontinued [Everlight Elec 19-217/BHC-ZL1M2RY/3T](https://www.lcsc.com/product-detail/Light-Emitting-Diodes-LED_Everlight-Elec-19-217-BHC-ZL1M2RY-3T_C72041.html)
might be a subtitute?: [Semicon SMLEN3BC8TT86](https://www.lcsc.com/product-detail/Light-Emitting-Diodes-LED_ROHM-Semicon-SMLEN3BC8TT86_C510296.html)

**Comment by androda** [comment]
Hello,

It looks like the 74LVTH125MTCX will work as an alternate for the SN74LVTH125PWR.  It hasn't been specifically tested as an alternate, but the datasheets seem to match.

As for the LED, I don't know how accurate that discontinuation notice is.  Was able to order 3000 of them at JLC just fine.  And the substitute you mention does look like it would drop right in.  Same size, same average forward voltage.

**Comment by konarik** [comment]
I have used 19-217/BHC-ZL1M2RY/6T for D3. According to datasheet it should be very similar LED (same package) only it seems to have optimized recommended PCB layout. LEDs works OK. Other parts were available during my order last month.
